src/backend/aspen/auth/auth0_management.py

```diff
@@ -84,21 +84,9 @@ def get_org_user_roles(self, org_id: str, user_id: str) -> List[str]:
         res = self.client.organizations.all_organization_member_roles(org_id, user_id)
         return [item["name"] for item in res]
 
-    @cache
-    def get_org_by_name(self, org_name: str) -> Auth0Org:
```

Comment:
This method seemed kinda dangerous since we don't have much of a unique constraint on group names, and we're already storing auth0 group ID's in the db. I refactored all the callers to use `get_org_by_id` instead.